### PR TITLE
Dynamic file size plus major code change

### PR DESCRIPTION
**Pull request by min050820**
이 포크는 다이나믹한 파일 사이즈를 받을 수 있으며, 4kb이상의 파일도 처리 가능합니다. 또한 코드도 많이 바뀌었습니다.

**이 코드는 컴파일을 한번도 하지 않았으니 병합 전에 테스트를 해봐야 합니다.**

### CODE REVIEW

**Comment by min050820** [comment]
머지는 언재하나요;;;

전 MSVC가 없는 <strike>데다가 귀찮아서</strike> <b>컴파일은 못합니다</b>

File: new-encrypt-program/main.cpp [hunk 1]
```diff
@@ -1,41 +1,77 @@
...
+		outputfile = buf;
+	}
+	if(outputfile == "")
+		outputfile = inputfile + ".enc";
+	
+	//Input
+	FILE * r, w;
+	r = fopen(inputfile.c_str(), "rb");
+	if(r==0) {
+		cerr << "Unable to open input file!" << endl;
```
Comment (spear-root) [inline new-encrypt-program/main.cpp]: What is cerr()??
@min050820

**Comment by min050820** [comment]
A std::ostream that points to stderr. It's basically a cout that is used to print errors.
<strike>Also, can you google before asking?</strike>

File: new-encrypt-program/main.cpp (hunk 1 above)
Comment (min050820) [inline new-encrypt-program/main.cpp]: A std::ostream that points to stderr. It's basically a cout that is used to print errors.
<i><strike>Also, can you google before asking?</strike></i>

**Comment by min050820** [comment]
You'll probably want to move those strings to some other file along with a function to manage interactive session.

**Comment by spear-root** [comment]
What strings?

**Comment by spear-root** [comment]
Let's go to Slack.

**Comment by min050820** [comment]
@spear-root What strings? Interactive UI messages, of course!

**Comment by min050820** [comment]
Closed because nobody could merge it. I told him to contact GitHub, but he
wouldn't listen. XD

P.S. I'll code some changes and it hopefully will improve UI, just like
what he said.
P.P.S. I hate UI coding 'cuz it involves a lot of
duplicative-but-not-exactly-same code.
P.P.P.S. Maybe I'm doing it wrong? Somebody please open an issue I'm doing
it wrong.

2017. 1. 31. 오후 11:57에 "spear-root" <notifications@github.com>님이 작성:

Closed #1 <https://github.com/spear-root/new-encrypt-program/pull/1>.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/spear-root/new-encrypt-program/pull/1#event-943176037>,
or mute the thread

**Pull request closed without merging**